examples/tools_support.ipynb

```diff
@@ -0,0 +1,7979 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "l68Urk97Wmut"
+      },
+      "source": [
+        "# 🛠️ Tools support\n",
+        "\n",
+        "<a href=\"https://colab.research.google.com/github/deepset-ai/haystack-experimental/blob/main/examples/tools_support.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\" width=\"200\"/></a>\n",
+        "\n",
+        "In this notebook, we introduce the experimental features we've been developing for tool/function calling support in Haystack.\n",
+        "\n",
+        "*Experimental means that these features are accessible in `haystack_experimental`, are open to feedback, and are subject to change.*\n",
+        "\n",
+        "- We refactored the `ChatMessage` dataclass, to be more flexible and future-proof.\n",
+        "- We introduced some new dataclasses: `ToolCall`, `ToolCallResult`, and `Tool`.\n",
+        "- We added support for tools in the `OpenAIChatGenerator` (and more to come).\n",
+        "- We introduced the `ToolInvoker` component, to actually execute tool calls prepared by Language Models."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "toc",
+        "id": "sxtzQK0CWubE"
+      },
+      "source": [
```

Comment:
Remove These TOCs that have `>> ` at the beginning because they will look weird in the cookbooks on the website

Reply:
done